plugins/redirect/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
If this is required, it should probably be scaffolded in with the previous details, or deleted